jbpm-wb-forms/jbpm-wb-forms-backend/src/main/java/org/jbpm/workbench/forms/display/backend/FormRendererProxyServlet.java

```diff
@@ -107,8 +112,28 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
             String processId = extractValue(processIdPattern, pathInfo);
             ProcessServicesClient processServicesClient = getClient(serverTemplateId,                                                      
                                                                     ProcessServicesClient.class);
-            
-            Long responseBody = processServicesClient.startProcess(containerId, processId, new StringContentMap(body));
+            Long responseBody = 0L;
+            if (correlationKey != null && !correlationKey.isEmpty()) {
+
+                responseBody = processServicesClient.startProcess(containerId, processId, new CorrelationKey() {
```

Comment:
such inline class seems suspicious, I don't have the insights, but wouldn't be for example `org.jbpm.workbench.pr.backend.server.model.RemoteCorrelationKey` usable here? @pefernan wdyt?

Reply:
@jstastny-cz @bxf12315 I think it could work fine here.

Reply:
@pefernan just to make sure, which could work better, the inline class or RemoteCorrelationKey class? :-)

Reply:
It's like this in main, resolving the thread.